src/coreclr/inc/ex.h

```diff
@@ -811,14 +808,10 @@ Exception *ExThrowWithInnerHelper(Exception *inner);
     {                                                                                   \
         STATETYPE               __state STATEARG;                                       \
         typedef DEFAULT_EXCEPTION_TYPE  __defaultException_t;                           \
-        SCAN_EHMARKER();                                                                \
         PAL_CPP_TRY                                                                     \
         {                                                                               \
-            SCAN_EHMARKER_TRY();                                                        \
-            SCAN_EHMARKER();                                                            \
             PAL_CPP_TRY                                                                 \
             {                                                                           \
-                SCAN_EHMARKER_TRY();                                                    \
                 CAutoTryCleanup<STATETYPE> __autoCleanupTry(__state);                   \
                 /* prevent annotations from being dropped by optimizations in debug */  \
                 INDEBUG(static bool __alwayszero;)                                      \
```

Comment:
We can remove these debug-only volatile statics. They're onnly here to enforce that the annotation calls in the `SCAN_*` macros weren't dropped,